templates/storage-management-policy.json

```
@@ -0,0 +1,33 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "storageAccountName": {
+      "type": "string",
+      "metadata": {
+        "description": "Name of the storage account"
+      }
+    },
+    "policyRules": {
+      "type": "array",
+      "defaultValue": [],
+      "metadata": {
+        "description": "A list of management policy rules"
+      }
+    }
+  },
+  "variables": {},
+  "resources": [
+    {
+      "name": "[concat(parameters('storageAccountName'), '/default')]",
+      "type": "Microsoft.Storage/storageAccounts/managementPolicies",
+      "apiVersion": "2019-04-01",
```

```suggestion
      "apiVersion": "2019-06-01",
```
Can be latest https://docs.microsoft.com/en-us/azure/templates/microsoft.storage/storageaccounts/managementpolicies